CHANGELOG.md

```diff
@@ -3,9 +3,11 @@
 [Full changelog](https://github.com/mozilla/glean/compare/v57.0.0...main)
 
 * General
-  * Update `glean_parser` to v12.0.1 ([release notes](https://github.com/mozilla/glean_parser/releases/tag/v12.0.0))
+  * Update `glean_parser` to v12.0.1 ([release notes](https://github.com/mozilla/glean_parser/releases/tag/v12.0.1))
+  * Update `glean_parser` to v13.0.0 ([release notes](https://github.com/mozilla/glean_parser/releases/tag/v13.0.0))
```

Comment:
I think I would prefer to update the existing line to show the version in the release rather than having two separate glean_parser versions listed here.

Reply:
okily dokily